bigquery/samples/getting_started.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+# Copyright 2015, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sample for making BigQuery queries using the python sdk.
+
+This is a command-line script that queries the public wikipedia-revision
+dataset, and displays the 10 most revised article titles.
+"""
+from pprint import pprint
+
+from apiclient.discovery import build
+from apiclient.errors import HttpError
+
+from oauth2client.client import GoogleCredentials
+
+
+# The id of the project to run queries under.
+PROJECT_ID = 'bigquery-devrel-samples'
+
+
+def main():
+    # [START build_service]
+    # Grab the application's default credentials from the environment.
```

Comment:
I feel like that's better left in the doc that uses this sample - urls take up a lot of room, and warrant some text around it explaining why you'd want to go there. eg if I were to add a url in this comment, I'd also want to say something like, "For more information on application default credentials, go to: http...", and that's another two lines of code. If I could just add a footnote to ADC, or link it, that'd work, but since I can't, I'd rather keep the code brief and to the point.